### ISSUE TITLE
Requirements::block not respecting resource path

### ISSUE THREAD

**Issue by flashbackzoo** [silverstripe/silverstripe-framework#7856]
## Affected Version

4.0.1

## Description

I came across this when trying to block a module script, specifically [silverstripe/tagfield](https://github.com/silverstripe/silverstripe-tagfield).

I expected this to work because it's [how the module requires the script](https://github.com/silverstripe/silverstripe-tagfield/blob/master/src/TagField.php#L204)

```php
Requirements::block('silverstripe/tagfield:js/TagField.js');
```

I also tried the resource URL

```php
Requirements::block('resources/silverstripe/tagfield/js/TagField.js');
```

Turns out you have to use the vendor path

```php
Requirements::block('vendor/silverstripe/tagfield/js/TagField.js');
```

Is that expected behaviour?

**Comment by robbieaverill** [comment]
Thanks for reporting!

We fixed this in 4.0.2.

See: https://github.com/silverstripe/silverstripe-framework/issues/7708

Cheers =)

**Comment by flashbackzoo** [comment]
Ahh - thanks!
